builtins/amp-img.js

```diff
@@ -221,51 +167,33 @@ export class AmpImg extends BaseElement {
   }
 
   /**
-   * @return {!Promise}
    * @private
    */
-  updateImageSrc_() {
-    if (this.getLayoutWidth() <= 0) {
-      return Promise.resolve();
-    }
-
-    if (!this.useNativeSrcset_) {
-      const src = this.srcset_.select(
-          // The width should never be 0, but we fall back to the screen width
-          // just in case.
-          this.getViewport().getWidth() || this.win.screen.width,
-          this.getDpr());
-      if (src == this.img_.getAttribute('src')) {
-        return Promise.resolve();
-      }
-
-      this.img_.setAttribute('src', src);
+  hideFallbackImg_() {
```

Comment:
This logic is directly taken from the then block of then original `updateImageSrc` and should be unchanged.